### PR TITLE
ospdo adoption makefile tests

### PR DESCRIPTION
**Pull request by pinikomarov**
Add Director Operator test execution scripts for adoption

Related CI patch and background:
https://gitlab.cee.redhat.com/ci-framework/ci-framework-jobs/-/merge_requests/966

PRs submitted together :
[related PR list](https://issues.redhat.com/browse/OSPRH-2984?focusedId=25950569&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25950569)

Jiras:
https://issues.redhat.com/browse/OSPRH-1891
https://issues.redhat.com/browse/OSPRH-2984

### CODE REVIEW

**Review by jistr — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -29,6 +29,16 @@ test-minimal:  ## Launch minimal test suite
 	mkdir -p tests/logs
 	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) $(TEST_ARGS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
 
+test_ospdo_pre_dataplane: TEST_OUTFILE := tests/logs/test_minimal_out_$(shell date +%FT%T%Z).log
```
Comment (jistr) [inline Makefile:32]: Could also be called 'test_ospdo_controlplane' but i'm just nitpicking. It's good to merge IMO.

**Comment by pinikomarov** [comment]
recheck

**Comment by jistr** [comment]
recheck

**Comment by pinikomarov** [comment]
recheck

**Comment by pinikomarov** [comment]
recheck

**Review by jistr — APPROVED** [review]

**Comment by jistr** [comment]
Approved since last week, merging.

**Pull request merged**